src/main/java/com/io/gittracker/controllers/MainViewPresenter.java

```diff
@@ -81,9 +95,14 @@ private void clearTileList() {
 
     private VBox createTile(Repository repo) {
         VBox tile = new VBox();
+        tile.getStyleClass().add("repoTile");
         HBox upperRow = new HBox();
         HBox lowerRow = new HBox();
         Label repoName = new Label(repo.getGithubName());
```

Comment:
With this change the link button is not really visible for the user, consider adding styles to :hover and :pressed pseudo-classes.

Also cursor HAND would look nice here.

Alternatively there is an already styled Hyperlink class from JavaFX.